ang/civicase/ActivityFeed.js

```diff
@@ -432,25 +425,29 @@
        */
       function affixActivityDetailsPanel () {
         $timeout(function () {
-          $activityDetailsPanel = $element.find('.panel');
+          $activityDetailsPanel = $element.find('.civicase__activity-panel');
           $filter = $('.civicase__activity-filter');
           $feedListContainer = $('.civicase__activity-feed__list-container');
           $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
           $toolbarDrawer = $('#toolbar');
 
           $activityDetailsPanel.affix({
             offset: {
-              top: $element.find('.panel').offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
+              top: $activityDetailsPanel.offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
               bottom: $($document).height() - ($feedListContainer.offset().top + $feedListContainer.height())
             }
-          }).on('affixed.bs.affix', function () {
+          });
+
+          $activityDetailsPanel.on('affixed.bs.affix', function () {
             $activityDetailsPanel
               .css('top', ($toolbarDrawer.height() + $tabs.height() + $filter.height()))
```

Comment:
why do we need to define `top` and `padding-top` at all right after the element has been affixed? Shouldn't the `offset.top` property given to `affix()` be enough to define the top offset?

Reply:
No it does not. Affix lib does not set the positioning, and the offset needs to be mentioned manually.
For example: see this example https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_scrollspy_affix2&stacked=h